main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 	config, clientSet := service.Init(cfg)
 	client.ClientSet = clientSet
 
-	common.PrintInfo(client.ClientSet, config)
+	common.PrintInfo(client.ClientSet, config, cfg)
```

Comment:
I'm not sure this pattern is quite as clean. It seems redundant to pass an object created from `common.InitArgs` back into `common.PrintInfo`. I think it's cleaner to set a default global 
```
var ConformanceImage string
``` 
in common. Then explicitly overwrite it in `InitArgs` if it's set:

```
ConformanceImage = cfg.ConformanceImage
```

Then we can check in `PrintInfo` if it's empty, or set it like:

```
  if ConformanceImage == "" {
      ConformanceImage = "registry.k8s.io/conformance:" + serverVersion.String()
  }
  ```
  
  By doing this we'll also need to make sure that `service.RunE2E` also is updated to reference `common.ConformanceImage` instead of `cfg.ConformanceImage`.